sidebars.js

```diff
@@ -132,6 +132,7 @@ module.exports = {
             "field-types/formula",
           ],
         },
+        "workflows/field-names",
```

Comment:
@BeeBombshell can this be changed to the database section as field name/key is related to columns